1-exercises/A-accessing-values/exercise1.js

```diff
@@ -5,19 +5,20 @@
 */
 
 let dog = {
-  breed: "Dalmatian",
-  name: "Spot",
-  isHungry: true,
-  happiness: 6
+  breed: "Dalmatian",//string
+  name: "Spot", //string
+  isHungry: true,//boolean
+  happiness: 6 //number
 };
 
 /*
   You can access the values of each property using dot notation.
   Log the name and breed of this dog using dot notation.
 */
 
-let dogName; // complete the code
-let dogBreed; // complete the code
+let dogName = "Spot"
```

Comment:
You should use the dot notation here  the answer should be let dog.Name = "Spot"